src/components/searchbar/searchbar.ts

```diff
@@ -63,6 +63,7 @@ export class Searchbar extends BaseInput<string> {
   _isActive: boolean = false;
   _showCancelButton: boolean = false;
   _animated: boolean = false;
+  _isRTL: boolean = this._plt.isRTL();
```

Comment:
`isRTL` is now a variable of `platform`, and not a method. so instead of having, this line, and then `if (this._isRTL)`, use:
`if (this._plt.isRTL)`

It will then also work on realtime direction switch.

Nice work!